src/test/test-execute.c

```diff
@@ -233,67 +227,77 @@ static void test_exec_readonlypaths(Manager *m) {
 static void test_exec_readwritepaths(Manager *m) {
 
         if (path_is_read_only_fs("/") > 0)
+                log_notice("Root directory is readonly, skipping %s", __func__);
                 return;
```

Comment:
this needs to be enclosed in {} now, since it contains two lines now

Reply:
Ouch! Updated.